### ISSUE TITLE
出来的是噪声图，怎么回事啊？

### ISSUE THREAD

**Issue by reitani** [mit-han-lab/ComfyUI-nunchaku#104]
![Image](https://github.com/user-attachments/assets/0e975a55-ca33-482c-8511-89978397b5f6)

我公司和家里两台电脑，按照同样的步骤安装之后，一台好用，提速4倍左右，另一台出来的就是上面这种噪声图。哪位能给指点一下吗？谢谢。
nunchaku 2.0   pytorch 2.6   python 3.12

**Comment by zzttqu** [comment]
I have the same problem

**Comment by ita9naiwa** [comment]
Could you let me know your setup more specifically?

e.g., 
- Are you using windows or mac?
- Which GPU and CUDA version are you using?

**Comment by reitani** [comment]
> Could you let me know your setup more specifically?
> 
> e.g.,
> 
> * Are you using windows or mac?
> * Which GPU and CUDA version are you using?


I don't know much about device,
is this piece of log from my comfy ui work?

** ComfyUI startup time: 2025-04-15 08:34:26.361555
** Platform: Windows
** Python version: 3.12.7 (tags/v3.12.7:0b05ead, Oct  1 2024, 03:06:41) [MSC v.1941 64 bit (AMD64)]
** Python executable: C:\Users\UJAE115\Desktop\comfy\ComfyUI_windows_portable\python_embeded\python.exe
** ComfyUI Path: C:\Users\UJAE115\Desktop\comfy\ComfyUI_windows_portable\ComfyUI
** Log path: C:\Users\UJAE115\Desktop\comfy\ComfyUI_windows_portable\comfyui.log
[ComfyUI-Manager] WARN: Unsafe - SSL verification bypass option is Enabled. (see ComfyUI-Manager/config.ini)

Prestartup times for custom nodes:
   0.0 seconds: C:\Users\UJAE115\Desktop\comfy\ComfyUI_windows_portable\ComfyUI\custom_nodes\rgthree-comfy
   0.0 seconds: C:\Users\UJAE115\Desktop\comfy\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Easy-Use
   3.4 seconds: C:\Users\UJAE115\Desktop\comfy\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager

Total VRAM 8188 MB, total RAM 65168 MB
pytorch version: 2.6.0+cu124
Set vram state to: NORMAL_VRAM
Device: cuda:0 NVIDIA RTX 3000 Ada Generation Laptop GPU : cudaMallocAsync

**Comment by aoleg** [comment]
I have the same problem. SwarmUI, torch-2.8.0.dev20250404+cu128.dist-info, torchvision-0.22.0.dev20250405+cu128.dist-info, Python 3.12.9
triton_windows-3.3.0a0.post17.dist-info
RTX 5080
Models tried: svdq-fp4-flux.1-dev and svdq-fp4-shuttle-jaguar

**Comment by ita9naiwa** [comment]
thanks,
I find it difficult to reproduce
could someone share workflow.json to reproduce??

**Comment by fiattt** [comment]
[test_workflow.json](https://github.com/user-attachments/files/19773163/test_workflow.json)  

> thanks, I find it difficult to reproduce could someone share workflow.json to reproduce??

**Comment by aoleg** [comment]
> thanks, I find it difficult to reproduce could someone share workflow.json to reproduce??

Here's my result: https://imgur.com/a/RdixzLe
Debug logs: https://paste.denizenscript.com/View/132676

I am using SwarmUI in the following configuration:

- SwarmUI 0.9.6
- Latest version of comfy portable (Windows), just downloaded from comfy Releases
- Blackwell (RTX 5080) GPU, so updated Torch to exactly the following (newer builds don't work): https://download.pytorch.org/whl/nightly/cu128/torch-2.8.0.dev20250404%2Bcu128-cp312-cp312-win_amd64.whl https://download.pytorch.org/whl/nightly/cu128/torchvision-0.22.0.dev20250405%2Bcu128-cp312-cp312-win_amd64.whl
https://download.pytorch.org/whl/nightly/cu128/torchaudio-2.6.0.dev20250405%2Bcu128-cp312-cp312-win_amd64.whl
- Python 3.12.9 (which is preinstalled with current comfy portable)
- Nunchaku wheel from huggingface: nunchaku-0.2.0+torch2.8

**Comment by aoleg** [comment]
Update: I have now updated the backend to https://github.com/comfyanonymous/ComfyUI/releases/download/latest/ComfyUI_windows_portable_nvidia_or_cpu_nightly_pytorch.7z
Unfortunately still having exactly the same behavior (this backend uses Python 3.13 by the way, so at least we can rule out Python). It seems likely that the current Nunchaku wheel does not support Blackwell/fp4 quants properly,

**Comment by aoleg** [comment]
According to https://github.com/mcmonkeyprojects/SwarmUI/issues/711 , this is an issue in nunchaku or nunchaku-comfy related specifically to Blackwell and fp4.

**Comment by edisonchan** [comment]
Do not use --gpu-only.

**Comment by aoleg** [comment]
I don't use --gpu-only (assuming it's a backend command line argument), still have the issue.

**Comment by zzttqu** [comment]
![Image](https://github.com/user-attachments/assets/120f8cfc-207a-47f9-b745-8aebfefd38a3)
这个是我的工作流和生成的噪点图，我使用了官方的example中的dev工作流，只调整了步数并删除了lora。
我使用的是comfyui的官方整合包，下边是我的设备信息。
Total VRAM 8192 MB, total RAM 16049 MB
pytorch version: 2.6.0+cu126
xformers version: 0.0.29.post3
Set vram state to: NORMAL_VRAM
Device: cuda:0 NVIDIA GeForce RTX 3070 Ti Laptop GPU : cudaMallocAsync
Using xformers attention
Python version: 3.12.9 (main, Feb 12 2025, 14:52:31) [MSC v.1942 64 bit (AMD64)]
ComfyUI version: 0.3.29
下边是我的虚拟环境中的pip list信息
Package                      Version
---------------------------- ------------------
absl-py                      2.2.2
accelerate                   1.6.0
addict                       2.4.0
aggdraw                      1.3.19
aiohappyeyeballs             2.4.6
aiohttp                      3.11.12
aiosignal                    1.3.2
albucore                     0.0.16
albumentations               1.4.15
annotated-types              0.7.0
antlr4-python3-runtime       4.9.3
anyio                        4.9.0
argostranslate               1.9.6
attrs                        25.1.0
av                           14.2.0
beautifulsoup4               4.13.3
bidict                       0.23.1
bitsandbytes                 0.45.5
blend-modes                  2.2.0
blind-watermark              0.4.4
cachetools                   5.5.2
certifi                      2025.1.31
cffi                         1.17.1
chardet                      5.2.0
charset-normalizer           3.4.1
click                        8.1.8
clip-interrogator            0.6.0
cmake                        4.0.0
color-matcher                0.6.0
colorama                     0.4.6
coloredlogs                  15.0.1
colorlog                     6.9.0
colour-science               0.4.6
comfyui-workflow-templates   0.1.1
contourpy                    1.3.1
cryptography                 44.0.0
cssselect2                   0.8.0
cstr                         0.1.0
ctranslate2                  4.6.0
cycler                       0.12.1
ddt                          1.7.2
deep-translator              1.11.4
deprecated                   1.2.18
diffusers                    0.33.1
dill                         0.4.0
distro                       1.9.0
docopt                       0.6.2
docutils                     0.21.2
easydict                     1.13
einops                       0.8.0
eval-type-backport           0.2.2
fairscale                    0.4.13
fastapi                      0.115.12
ffmpy                        0.3.0
filelock                     3.17.0
flatbuffers                  25.2.10
flet                         0.27.6
fonttools                    4.57.0
frozenlist                   1.5.0
fsspec                       2025.2.0
ftfy                         6.3.1
fvcore                       0.1.5.post20221221
gdown                        5.2.0
gguf                         0.14.0
gitdb                        4.0.12
gitpython                    3.1.44
google-ai-generativelanguage 0.6.15
google-api-core              2.24.2
google-api-python-client     2.167.0
google-auth                  2.39.0
google-auth-httplib2         0.2.0
google-genai                 1.10.0
google-generativeai          0.8.4
googleapis-common-protos     1.70.0
googletrans-py               4.0.0
gputil                       1.4.0
grpcio                       1.71.0
grpcio-status                1.71.0
h11                          0.14.0
h2                           4.2.0
hpack                        4.1.0
httpcore                     1.0.8
httplib2                     0.22.0
httpx                        0.28.1
huggingface-hub              0.28.1
humanfriendly                10.0
hydra-core                   1.3.2
hyperframe                   6.1.0
idna                         3.10
imageio                      2.37.0
img2texture                  1.0.6
importlib-metadata           8.6.1
iopath                       0.1.10
jax                          0.5.3
jaxlib                       0.5.3
jinja2                       3.1.5
jiter                        0.9.0
joblib                       1.4.2
jsonpatch                    1.33
jsonpointer                  3.0.0
jsonschema                   4.23.0
jsonschema-specifications    2024.10.1
kiwisolver                   1.4.8
kornia                       0.8.0
kornia-rs                    0.1.8
lark                         1.2.2
lazy-loader                  0.4
llvmlite                     0.44.0
loguru                       0.7.3
lxml                         5.3.2
manifold3d                   3.0.1
mapbox-earcut                1.0.3
markdown-it-py               3.0.0
markupsafe                   3.0.1
matplotlib                   3.10.1
matrix-client                0.4.0
mdurl                        0.1.2
mediapipe                    0.10.21
ml-dtypes                    0.5.1
mpmath                       1.3.0
mss                          10.0.0
multidict                    6.1.0
networkx                     3.4.2
numba                        0.61.2
numpy                        1.26.4
nunchaku                     0.2.0+torch2.6
oauthlib                     3.2.2
omegaconf                    2.3.0
onnxruntime                  1.21.0
open-clip-torch              2.32.0
openai                       1.74.0
opencv-contrib-python        4.11.0.86
opencv-python                4.11.0.86
opencv-python-headless       4.11.0.86
opt-einsum                   3.4.0
packaging                    24.2
pandas                       2.2.3
peft                         0.15.1
piexif                       1.1.3
pilgram                      1.2.1
pillow                       11.1.0
pip                          24.3.1
pixeloe                      0.1.4
platformdirs                 4.3.7
pooch                        1.8.2
portalocker                  3.1.1
propcache                    0.2.1
proto-plus                   1.26.1
protobuf                     5.29.4
psd-tools                    1.10.7
psutil                       6.1.1
py-cpuinfo                   9.0.0
pyasn1                       0.6.1
pyasn1-modules               0.4.2
pycollada                    0.9
pycparser                    2.22
pydantic                     2.11.3
pydantic-core                2.33.1
pygithub                     2.5.0
pygments                     2.19.1
pyjwt                        2.8.0
pymatting                    1.1.13
pynacl                       1.5.0
pyparsing                    3.2.3
pyreadline3                  3.5.4
pysocks                      1.7.1
python-dateutil              2.9.0.post0
python-engineio              4.12.0
python-socketio              5.13.0
pytz                         2025.2
pywavelets                   1.8.0
pywin32                      310
pyyaml                       6.0.2
pyzbar                       0.1.9
qrcode                       8.1
referencing                  0.36.2
regex                        2024.11.6
rembg                        2.0.65
repath                       0.9.0
reportlab                    4.3.1
requests                     2.32.3
rich                         13.9.4
rpds-py                      0.24.0
rsa                          4.9.1
rtree                        1.4.0
sacremoses                   0.0.53
safetensors                  0.5.2
scikit-image                 0.25.2
scikit-learn                 1.6.1
scipy                        1.15.1
seaborn                      0.13.2
segment-anything             1.0
sentencepiece                0.2.0
setuptools                   78.1.0
shapely                      2.1.0
shellingham                  1.5.4
simple-websocket             1.1.0
tinycss2                     1.4.0
tokenizers                   0.21.0
toml                         0.10.2
torch                        2.6.0+cu126
torchaudio                   2.6.0+cu126
torchsde                     0.2.6
torchvision                  0.21.0+cu126
tqdm                         4.67.1
trampoline                   0.1.2
transformers                 4.48.3
transparent-background       1.3.3
trimesh                      4.6.6
typer                        0.15.1
typer-config                 1.4.2
typing-extensions            4.12.2
typing-inspection            0.4.0
tzdata                       2025.2
ultralytics                  8.3.109
ultralytics-thop             2.0.14
uritemplate                  4.1.1
urllib3                      1.26.20
uv                           0.5.29
vhacdx                       0.0.8.post2
wcwidth                      0.2.13
webencodings                 0.5.1
websockets                   15.0.1
wget                         3.2
win32-setctime               1.2.0
wrapt                        1.17.2
wsproto                      1.2.0
xformers                     0.0.29.post3
xxhash                       3.5.0
yacs                         0.1.8
yapf                         0.43.0
yarl                         1.18.3
zhipuai                      2.1.5.20250415
zipp                         3.21.0

**Comment by fiattt** [comment]
`export NUNCHAKU_LOAD_METHOD="READNOPIN"`
ok了  加载模型不会再耗费1分钟多的时间了
